test/e2e/middleware-general/app/middleware.js

```diff
@@ -1,5 +1,5 @@
-/* global globalThis, URLPattern */
-import { NextRequest, NextResponse } from 'next/server'
+/* global globalThis */
+import { NextRequest, NextResponse, URLPattern } from 'next/server'
```

Comment:
Can we add this in TypeScript middleware test suite to ensure the types are correct as well? https://github.com/vercel/next.js/blob/canary/test/production/middleware-typescript/app/middleware.ts